src/components/Standings/Standings.js

```diff
@@ -5,85 +5,125 @@ import {
+    })
+    
+    // Get the team number and user ID of the current user.
+    // These are for highlighting the user's position in the teams and users scoreboards
+    this.props.firebase.checkAuthUser(user => {
+      if(user){
+        this.props.firebase.getUserData(user.uid).then(data => {
+          this.setState({userTeamNumber: data.data().team})
+          this.setState({userID: data.data().uid})
+        })
+      }
     })
   }
 
   render () {
-    //   places renders the Place object for each team in the correct order
-    const places = this.state.allTeams
-      .slice(0, this.state.shownNumber)
-      .map((team, index) => (
-        <Place
-          rank={index + 1}
-          teamName={team.name}
-          teamNumber={team.number}
-          points={team.points}
-          key={team.id}
-        />
-      ))
+    // Creates places list for teams, which renders Place object for each team in order
+    if (this.state.showScoresByTeam) {
+      var places = (this.state.showPointsPerMember ?  this.state.allTeamsPPM : this.state.allTeams)
+        .slice(0, this.state.shownNumber)
+        .map((team, index) => (
+          <Place
+            rank={index + 1}
+            teamName={team.name}
+            teamNumber={team.number}
+            points={team.points}
+            key={team.id}
+            size={team.size}
+            userTeamNumber={this.state.userTeamNum}
+            showPointsPerMember={this.state.showPointsPerMember}
+            showScoresByTeam={this.state.showScoresByTeam}
+            pointsPerMember={Math.floor(team.points / team.size * 10) / 10}
+          />
+        ))
+    } 
+    // Creates places list for people, which renders Place object for each user in order
+    else {
+      var places = this.state.allUsers
+        .slice(0, this.state.shownNumber)
+        .map((user, index) => (
+          <Place
+            rank={index + 1}
+            teamName={user.name}
+            teamNumber={user.team}
+            points={user.points}
+            key={user.uid}
+            size={1}
+            userTeamNumber={this.state.userTeamNum}
+            uid={user.uid}
+            userID={this.state.userID}
+            showPerMember={this.state.showPointsPerMember}
+            pointsPerMember={user.points}
+          />
+        ))
+    }
+
+    // If scoreboard is showing people rather than teams,
+    // Show top 10 users on scoreboard, with current user at bottom (if they're below 10th).
+    if (!this.state.showScoresByTeam && places.length > 10) {
+      let userPlace = new Place
+      let userPlaceRank = undefined
+      for (var i = 0; i < places.length; i++) {
+        if (places[i].key == this.state.userID) {
+          userPlace = places[i]
+          userPlaceRank = i
+          break
+        }
+      }
+      places = places.slice(0, 10)
+      if (userPlaceRank > 10) {
+        places = places.concat([userPlace])
+      }
+    }
 
     return (
```

Comment:
I like how the switches are set up right now, but I feel like we need to make it more obvious that these are buttons and are clickable. Any ideas?